controllers/topics/index.js

```diff
@@ -158,7 +158,7 @@ const getTopics = async (req, res) => {
 
 const followTopicV2 = async (req, res) => {
   try {
-    const {name} = req.body;
+    const {name, with_system_message = false} = req.body;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new parameter `with_system_message` is introduced with a default value of `false`. This is good practice as it ensures backward compatibility with existing calls to this function that do not provide this argument.
```diff
-    const {name} = req.body;
+    const {name, with_system_message = false} = req.body;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->